### PR TITLE
feat: multi chain info page

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

File: packages/wagmi/src/chains.ts [hunk 1]
```diff
@@ -101,6 +103,28 @@ export const bsc: Chain = {
...
+  //   name: 'Binance Chain Native Token',
+  //   symbol: 'BNB',
+  //   decimals: 18,
+  // },
+  // multicall: {
+  //   address: '0x72dba3Fa54C73D9EDB493e9F4eDf884439B1eBC4',
+  //   blockCreated: 20455688,
+  // },
+}
+
```
Comment (0xjojoex) [inline packages/wagmi/src/chains.ts]: just use the `mainnet`

File: src/pages/info/index.tsx [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+import { useRouter } from 'next/router'
+import { ChainId } from '@pancakeswap/sdk'
 
 const InfoPage = () => {
+  const router = useRouter()
+  const { chainId, chainName } = router.query
+  if (chainId === ChainId.ETHEREUM.toString() && !chainName)
+    router.replace({ pathname: '/info/eth', query: '' }, undefined, { shallow: true })
+  else if (!chainName && chainId === ChainId.BSC.toString())
+    router.replace({ pathname: '/info', query: '' }, undefined, { shallow: true })
```
Comment (0xjojoex) [inline src/pages/info/index.tsx]: can get the chainId and useEffect once

File: src/views/Info/components/CurrencyLogo/index.tsx [hunk 3]
```diff
@@ -16,15 +16,18 @@ export const CurrencyLogo: React.FC<
...
     size?: string
+    chainName?: 'ETH' | 'BSC'
   }>
-> = ({ address, size = '24px', ...rest }) => {
+> = ({ address, size = '24px', chainName = 'BSC', ...rest }) => {
   const src = useMemo(() => {
     const checksummedAddress = isAddress(address)
     if (checksummedAddress) {
-      return `https://assets-cdn.trustwallet.com/blockchains/smartchain/assets/${checksummedAddress}/logo.png`
+      return `https://assets-cdn.trustwallet.com/blockchains/${
```
Comment (0xjojoex) [inline src/views/Info/components/CurrencyLogo/index.tsx]: getTokenLogoURL

File: src/pages/info/index.tsx (hunk 2 above)
Comment (0xjojoex) [reply src/pages/info/index.tsx]: useActiveChainId or useActiveWeb3React
`sesssionChainId -> queryChainId -> wallet.chain`

File: src/state/info/queries/helpers.ts [hunk 4]
```diff
@@ -81,7 +151,7 @@ export const fetchChartData = async (
 
   while (!allFound) {
     // eslint-disable-next-line no-await-in-loop
-    const { data, error: fetchError } = await getEntityDayDatas(skip, address)
+    const { data, error: fetchError } = await getEntityDayDatas(chianName, skip, address)
```
Comment (ChefMomota) [inline src/state/info/queries/helpers.ts:154]: typo

File: src/views/Info/components/InfoTables/TransactionsTable.tsx [hunk 5]
```diff
@@ -98,10 +101,12 @@ const DataRow: React.FC<React.PropsWithChildren<{ transaction: Transaction }>> =
   const abs1 = Math.abs(transaction.amountToken1)
   const outputTokenSymbol = transaction.amountToken0 < 0 ? transaction.token0Symbol : transaction.token1Symbol
   const inputTokenSymbol = transaction.amountToken1 < 0 ? transaction.token0Symbol : transaction.token1Symbol
-
+  const chainName = useGetChainName()
   return (
     <ResponsiveGrid>
-      <LinkExternal href={getBlockExploreLink(transaction.hash, 'transaction')}>
+      <LinkExternal
+        href={getBlockExploreLink(transaction.hash, 'transaction', chainName === 'ETH' && ChainId.ETHEREUM)}
```
Comment (ChefMomota) [inline src/views/Info/components/InfoTables/TransactionsTable.tsx:108]: getBlockExploreLink(transaction.hash, 'transaction',  chainId)

File: src/views/Info/components/InfoTables/TransactionsTable.tsx [hunk 6]
```diff
@@ -117,7 +122,7 @@ const DataRow: React.FC<React.PropsWithChildren<{ transaction: Transaction }>> =
       <Text>
         <Text>{`${formatAmount(abs1)} ${transaction.token1Symbol}`}</Text>
       </Text>
-      <LinkExternal href={getBlockExploreLink(transaction.sender, 'address')}>
+      <LinkExternal href={getBlockExploreLink(transaction.sender, 'address', chainName === 'ETH' && ChainId.ETHEREUM)}>
```
Comment (ChefMomota) [inline src/views/Info/components/InfoTables/TransactionsTable.tsx:125]: getBlockExploreLink(transaction.hash, 'transaction',  chainId)

File: src/state/info/constant.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  [ChainId.ETHEREUM]: '/eth',
+}
+
+export const multiChainQueryClient = {
+  BSC: infoClient,
+  ETH: infoClientETH,
+}
+
+export const multiChainQueryEndPoint = {
+  BSC: INFO_CLIENT,
```
Comment (ChefMomota) [inline src/state/info/constant.ts:28]: Can we use chainId be the key?

File: src/utils/graphql.ts [hunk 8]
```diff
@@ -18,8 +18,31 @@ export const getGQLHeaders = (endpoint: string) => {
...
 
+export const infoClientETH = new GraphQLClient(INFO_CLIENT_ETH, { headers: getGQLHeaders(INFO_CLIENT_ETH) })
+
+export const infoServerClientETH = new GraphQLClient(INFO_CLIENT_ETH, {
+  headers: {
+    'X-Sf': process.env.SF_HEADER,
+  },
+  timeout: 5000,
+})
+
```
Comment (ChefMomota) [inline src/utils/graphql.ts:45]: Prefer using chainId to fetch no duplicate same function

**Pull request closed without merging**